src/Microsoft.Docs.LearnValidation/Models/ValidatorHierarchyItem.cs

```diff
@@ -17,6 +16,9 @@ public class ValidatorHierarchyItem : HierarchyItem, IValidateModel
 
         public bool IsDeleted { get; set; }
 
+        [JsonProperty("uid")]
+        public string UId { get; set; } = "";
+
         public string Uid => UId;
```

Comment:
If `UId` is marked as `[JsonProperty("uid")]`, is it equivalent to:

```suggestion
        public string Uid { get; set; } = "";
```

Reply:
we can't do this,because `IValidateModel` hava a `Uid` with `get` only